pkg/cvo/options/options.go

```diff
@@ -0,0 +1,8 @@
+package options
+
+type PromQLTarget struct {
+	QueryNamespace        string
+	QueryService          string
+	QueryCaBundleFile     string
+	BearerCredentialsFile string
+}
```

Comment:
Is there any reason you have moved this to a separate package? I might have guessed it to be part of `package clusterconditions`